### ISSUE TITLE
Many textarea

### ISSUE THREAD

**Issue by williansebastiao** [tiff/wysihtml5#543]
I have more of a textarea, and need to clear their contents in focus.
But only at first I can ... how do I get the other?

**Comment by IntellectProductions** [comment]
Do you have any code you can show us to see how you have things setup?

**Comment by williansebastiao** [comment]
Estou utilizando o seguinte código. 
Porém, o mesmo só executa no primeiro índice, no segundo ele não executa

editor.on("focus", function() {
    $(editor.textareaElement).parent().removeClass('has-error');
    $(editor.textareaElement).parent().find('span.help-block').detach();
});
